test/xpu_api/utilities/meta/meta.trans/meta.trans.other/invoke_result.pass.cpp

```diff
@@ -25,6 +25,7 @@
 #include "has_type_member.h"
 
 #if TEST_DPCPP_BACKEND_PRESENT
+#    if TEST_STD_VER == 17
```

Comment:
Why do we skip tests for `dpl::invoke_result`? It was not removed in C++20

Reply:
Fixed